<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AtmosphereDialog</class>
 <widget class="QDialog" name="AtmosphereDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>273</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Configure Atmosphere</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images/images.qrc">
    <normaloff>:/fromHelyx/newicons/icon.png</normaloff>:/fromHelyx/newicons/icon.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QLineEdit" name="density_lineEdit">
         <property name="font">
          <font>
           <family>Ubuntu</family>
           <pointsize>9</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Ubuntu</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Density [kg/m^3]:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Ubuntu</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Pressure [Pa]:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="velocity_lineEdit">
         <property name="font">
          <font>
           <family>Ubuntu</family>
           <pointsize>9</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Ubuntu</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Velocity [m/s]:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="pressure_lineEdit">
         <property name="font">
          <font>
           <family>Ubuntu</family>
           <pointsize>9</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="save_config_pushButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <family>Ubuntu</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string> Save Configuration</string>
         </property>
         <property name="icon">
          <iconset resource="../images/images.qrc">
           <normaloff>:/fromHelyx/fromHelyx/save16.png</normaloff>:/fromHelyx/fromHelyx/save16.png</iconset>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="accept_pushButton">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string> Accept</string>
       </property>
       <property name="icon">
        <iconset resource="../images/images.qrc">
         <normaloff>:/fromHelyx/fromHelyx/tick16.png</normaloff>:/fromHelyx/fromHelyx/tick16.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_pushButton">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string> Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../images/images.qrc">
         <normaloff>:/fromHelyx/fromHelyx/win_closeAll16.png</normaloff>:/fromHelyx/fromHelyx/win_closeAll16.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images/images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>accept_pushButton</sender>
   <signal>released()</signal>
   <receiver>AtmosphereDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel_pushButton</sender>
   <signal>released()</signal>
   <receiver>AtmosphereDialog</receiver>
   <slot>cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>cancel()</slot>
 </slots>
</ui>
